test-jest/cartesian/ReferenceArea.spec.tsx

```diff
@@ -116,11 +115,11 @@ describe('<ReferenceArea />', () => {
         <ReferenceArea y1={200} y2={300} fill="#666" alwaysShow />
       </BarChart>,
     );
-    expect(wrapper.find('.recharts-reference-area-rect').length).to.equal(1);
+    expect(container.querySelectorAll('.recharts-reference-area-rect')).toHaveLength(1);
   });
 
-  it('Render 1 line and 1 label when label is set to be a function in ReferenceArea', () => {
-    const renderLabel = props => {
+  test('Render 1 line and 1 label when label is set to be a function in ReferenceArea', () => {
+    const renderLabel = (props: any) => {
```

Comment:
nit: any way we can actually type these props?

Fine for the sake of the test migration though I'd like to avoid the use of `any` where possible